templates/layouts/_questionnaire.html

```diff
@@ -37,4 +37,12 @@
           })
       }}
   {% endblock %}
+
+  {% block cant_complete %}
+     {% if is_hub_enabled and are_hub_required_sections_complete and is_question %}
```

Comment:
This logic could be done in code and used to set `enable_cant_complete_link` template variable.

Reply:
Should this be called `enable_return_to_hub_link` since that is what this is doing and is only relevant when you have a hub?. @ajmaddaford

Reply:
Yeah, I would also swap `enable` for `display`, so `display_return_to_hub_link`

Reply:
We've updated them now. Thanks!